src/test/frontend/extensions/html_parser_test.cljs

```diff
@@ -0,0 +1,32 @@
+(ns frontend.extensions.html-parser-test
```

Comment:
❤️

Reply:
I just saw the tests failed. cljs support on node for hickory is still pretty new so things may not just work. https://github.com/clj-commons/hickory/pull/33 may have some useful tips

Reply:
Disabled the test for now because requiring jsdom (the trick) doesn't work.